apps/web/src/composables/users.ts

```diff
@@ -4,8 +4,8 @@ import useSSV from '@/composables/ssv'
 import useWallet from '@/composables/wallet'
 import { onMounted, ref } from 'vue'
 import { User } from '@casimir/types'
```

Comment:
```ts
import { User, AddAccountOptions, ...  } from '@casimir/types'
```

Reply:
Good catch and easy fix. I'm sure this is replicated in multiple places. Will clean up.